src/schemata/definitions/field.yml

```diff
@@ -8,14 +8,10 @@ allOf:
     properties:
       ui:widget:
         type: string
-        enum: ['checkbox', 'switch', 'textarea']
+        enum: ['checkbox', 'switch', 'textarea', 'input']
```

Comment:
comparing with [the list of widgets](https://github.com/rjsf-team/react-jsonschema-form/tree/v0.51.0/src/components/widgets), maybe we should use `text` instead of `input`?

Reply:
That naming and dep is not related, so I don't think that matters, but I don't mind the suggest change. Possibly adds more clarity. Wdyt @Abobos ?

Reply:
it does though.. since react-jsonschema-form is the one responsible for rendering the widget.. so the list of possible values must be something that  it understand..

Reply:
IIUC this addition is ignored, since we're [only checking for textarea](https://github.com/labforward/laboperator-server/blob/a4da2038e38c6c38ef8dfc8553eaddd3c5bf4150/app/javascript/react/components/forms/json_schema_form/fields/StringField.js)

Reply:
> That naming and dep is not related, so I don't think that matters, but I don't mind the suggest change. Possibly adds more clarity. Wdyt @Abobos?

I am fine with this, so far we update the description to match it.